packages/openapi-fetch/src/index.js

```diff
@@ -80,7 +80,7 @@ export default function createClient(clientOptions) {
       requestInit.body = bodySerializer(requestInit.body);
     }
     // remove `Content-Type` if serialized body is FormData; browser will correctly set Content-Type & boundary expression
-    if (requestInit.body instanceof FormData) {
+    if (requestInit.body && requestInit.body instanceof FormData) {
```

Comment:
```suggestion
    if (!requestInit.body || requestInit.body instanceof FormData) {
```

altho out of scope for this pr but we should delete the content type when body is empty, currently there's no proper way to remove the body and when I don't need to send any body to backend, backend throws content type json was used but no body provided

Reply:
Yeah I may have approved prematurely because I didn‘t realize at first this was attempting to solve 2 issues with 1 PR. My suggestions should be a ~2-line change in code, and a couple test additions to satisfy both.